### ISSUE TITLE
Accessibility Issues on Environments Window: Packages List View

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2228]
Missing properties from remove and upgrade buttons. 

![image](https://cloud.githubusercontent.com/assets/5177491/23570801/cbd8a1f2-001a-11e7-924a-0816553903d3.png)

**Comment by eclectir** [comment]
Error details-
MissingProperty - UIA_IsKeyboardFocusableProperty - Element cannot receive keyboard focus. The element might be reachable using the narrator but the user won't be able to invoke any action associated with the element using space or enter. On pressing the tab/arrow key,  the previous focused element will be used as the starting point. Even if the control can be invoked using other keyboard shortcuts, it is important for the control to be keyboard focusable because a user trying to invoke the control through a screen reader will accidentally invoke the previous control that has keyboard focus
https://msdn.microsoft.com/en-us/library/system.windows.automation.automationelement.automationelementinformation.iskeyboardfocusable(v=vs.110).aspx

**Comment by huguesv** [comment]
This has been fixed.
